From: Jason Rhinelander <jason@imaginary.ca>
Date: Fri, 17 Oct 2025 21:05:06 -0300
Subject: new boost compilation fix

---
 src/debug_utilities/object_sizes.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/debug_utilities/object_sizes.cpp b/src/debug_utilities/object_sizes.cpp
index d1b668c..8a8420a 100644
--- a/src/debug_utilities/object_sizes.cpp
+++ b/src/debug_utilities/object_sizes.cpp
@@ -70,7 +70,6 @@ int main(int argc, char* argv[]) {
 
     SL(boost::thread);
     SL(boost::asio::io_service);
-    SL(boost::asio::io_service::work);
     SL(boost::asio::steady_timer);
 
     SL(cryptonote::DB_ERROR);
