From: Jason Rhinelander <jason@imaginary.ca>
Date: Wed, 18 Dec 2024 22:44:08 -0400
Subject: allow older fmt

---
 external/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index d08c98d..65c7d8a 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -61,7 +61,7 @@ endmacro()
 
 system_or_submodule(OXENC oxenc liboxenc<1.1.0 oxen-encoding)
 system_or_submodule(OXENMQ oxenmq liboxenmq>=1.2.12 oxen-mq)
-system_or_submodule(FMT fmt fmt>=8.0.0 fmt)
+system_or_submodule(FMT fmt fmt fmt)
 
 add_subdirectory(db_drivers)
 add_subdirectory(easylogging++ easyloggingpp)
