From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 16 Dec 2024 12:26:14 -0400
Subject: Update fmt lib install path

---
 external/CMakeLists.txt | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index d08c98d..e13a556 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -59,9 +59,12 @@ macro(system_or_submodule BIGNAME smallname pkgconf subdir)
   endif()
 endmacro()
 
-system_or_submodule(OXENC oxenc liboxenc<1.1.0 oxen-encoding)
-system_or_submodule(OXENMQ oxenmq liboxenmq>=1.2.12 oxen-mq)
-system_or_submodule(FMT fmt fmt>=8.0.0 fmt)
+# deb: put our local version of fmt into the oxen-specific lib dir:
+set(FMT_LIB_DIR lib${OXEN_INSTALL_LIBDIR_SUFFIX} CACHE STRING "" FORCE)
+
+system_or_submodule(OXENC oxenc liboxenc>=1.1.0 oxen-encoding)
+system_or_submodule(OXENMQ oxenmq liboxenmq>=1.2.18 oxen-mq)
+system_or_submodule(FMT fmt fmt>=9.0.0 fmt)
 
 add_subdirectory(db_drivers)
 add_subdirectory(easylogging++ easyloggingpp)
